.github/workflows/idpay-openAPI-AppIO-diff.yml

```diff
@@ -0,0 +1,37 @@
+name: 'Check diff on idpay AppIO full swagger'
+
+on:
+  push:
```

Comment:
does this action triggers only after push (after PR merge), correct? Why not on PR opening?

Reply:
the check is performed every push, before the merge, only if there are changes in the folders underlying "domains/idpay-app"

Reply:
the action need to be executed in the context of a Pull Request, with `on.push` it runs even if no PR is opened or when a PR is merged. Does it sound correct to you?

I suggest using the `on.pull_request` or `on.pull_request_target` strategy in order to make it run only when a PR is opened or updated.

a snippet from the pr_title action: 
```
on:
  pull_request_target:
    types:
      - opened
      - edited
      - synchronize 
 ```

Reply:
the reasoning is correct